pkg/registry/core/service/ipallocator/metrics.go

```diff
@@ -71,7 +71,16 @@ var (
 			Help:           "Number of errors trying to allocate Cluster IPs",
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"cidr"},
+		[]string{"cidr", "allocation_mode"},
+	)
+	// clusterIPAllocatorInfo exposes the configuration details of the allocator.
+	clusterIPAllocatorInfo = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Name:           "allocator_info",
+			Help:           "A metric with a constant '1' value labeled by Service CIDR, the size of the allocation pool and the offset that define the start of the dynamic preferred allocation range",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"cidr", "allocator_size", "dynamic_range_offset"},
```

Comment:
what's the projected cardinality of these labels?

Reply:
max number of service cidrs is 2, so max 2

Reply:
What about once your OTHER kep lands?  Then this becomes kind of unbounded.  That seems bad.

Will we instead write this out to the status on those CCC resources?  Does that mean we abandon this metric?

Reply:
It is a good point and I will take a note about the OTHER kep, these metrics has to be migrated or _cidr_ should be bounded.
Today, all metrics takes `cidr` as a label, nad it should be safe because on how this is implemented in the registry, it is not possible to have more than 2 cidrs without a major refactor, and that refactor will require to remove all current allocators code